CMakeLists.txt

```diff
@@ -195,6 +196,13 @@ if(ENABLE_CALIPER)
     endif()
 endif(ENABLE_CALIPER)
 
+# Code instrumentation
+if(ENABLE_LIKWID)
+    find_package(likwid REQUIRED)
+    include_directories(${LIKWID_INCLUDE_DIR})
```

Comment:
The macro should be prefixe by `CORENEURON_` to avoid possible collision.

Reply:
could you please take a look at 3f280985ac66b40f152a558dcee97d374c40be42?

Reply:
That is very good! Thank you.
Please apply `cmake-format` utility (I think the variables list in the comments should be prefixed with '*' to prevent being reformated)

Reply:
We currently don't use `cmake-format` in this repository. So, I think this is quite alright with the latest changes :)

Reply:
That's fine then indeed.